mkdocs/docs/configuration.md

```diff
@@ -341,7 +341,7 @@ catalog:
 <!-- prettier-ignore-start -->
 
 !!! warning "Deprecated Properties"
-    `profile_name`, `region_name`, `botocore_session`, `aws_access_key_id`, `aws_secret_access_key`, `aws_session_token` are deprecated and will be removed in 0.8.0:
+    `profile_name`, `region_name`, `botocore_session`, `aws_access_key_id`, `aws_secret_access_key`, `aws_session_token` are deprecated and removed in 0.8.0:
```

Comment:
Since these properties are removed, shall we say they are "Removed Properties"?

We might also consider removing this warning (and the one below) and mention these removed properties in the release note, along with other removed properties/methods/....

Reply:
Changed to `Removed Properties`

I think its a good idea to leave it here, at least for this new release, so that people can look up why their old properties are no longer working 

![Screenshot 2024-10-31 at 9 25 13 AM](https://github.com/user-attachments/assets/4f4d448c-eb91-4569-a92f-9153bdb9a260)